src/components/DocumentFormular.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <section class="w-[70vw] max-w-[850px] min-w-[400px]" @keydown.esc="$emit('close')">
+    <header class="flex justify-center p-5 border-solid border-b border-vologray-200">
+      <h2 class="text-[20px] text-bold font-medium">
+        Neues Dokument für {{ volunteerStore.selectedVolunteer.person.firstname }}
+      </h2>
+    </header>
+    <main class="p-8">
+      <p
+        v-if="errorMessage"
+        class="flex justify-center text-sm font-light rounded border border-1 mb-4 p-3 bg-red-100 border-red-500 text-red-500"
+      >
+        🤷‍♀️ Upsi! Da ist wohl was schief gelaufen... Versuchs bitte nochmal.
+      </p>
+      <div v-if="!volunteerStore.fetching" class="flex justify-center">
+        <!-- left column -->
+        <div class="flex-1">
+          <p class="text-[13px] text-vologray-400 pe-20">
+            Lade ein neues Dokument für
+            {{ volunteerStore.selectedVolunteer.person.firstname }} hoch.
+          </p>
+        </div>
+        <!-- right column -->
+        <div class="flex-1">
+          <form class="flex flex-col gap-2" id="new-document" @submit.prevent="onSubmit" novalidate>
+            <FormularInput
+              name="name"
+              label="Name"
+              type="text"
+              id="name"
+              :required="true"
+              :hasError="validationErr.name"
+              v-model="formData.name"
+            />
+            <FormularSelectBox
+              :title="[
```

Comment:
this data has to be set dynamically by passing in the descriptions from the backend